src/stylesheets/elements/_typography.scss

```diff
@@ -184,76 +149,81 @@ dfn {
   p,
   ul:not(.usa-accordion):not(.usa-accordion-bordered),
   ol:not(.usa-accordion):not(.usa-accordion-bordered) {
-    max-width: $text-max-width;
+    max-width: measure($theme-measure-text);
   }
 }
 
 .usa-content-list {
-  max-width: $text-max-width;
-}
-
-.usa-sans {
-  p,
-  a,
-  li,
-  span {
-    font-family: $font-sans;
-  }
-
-  a {
-    border-bottom: none;
-    font-weight: $font-bold;
-  }
-}
-
-.usa-serif {
-  p,
-  a,
-  li,
-  span {
-    font-family: $font-serif;
-  }
+  max-width: measure($theme-measure-text);
 }
 
 .usa-display {
-  @include h3;
+  @include typeset-h3;
   margin-bottom: 0;
 
   @include media($small-screen) {
-    @include h1;
+    @include typeset-h1;
   }
 
   @include media($medium-screen) {
-    @include title;
+    @include typeset-title;
   }
 }
 
 .usa-font-lead {
-  font-family: $font-serif;
-  font-size: $lead-font-size;
-  font-weight: $font-normal;
-  line-height: $lead-line-height;
-  max-width: $lead-max-width;
+  @include typeset(
+    $theme-font-family-lead,
+    $theme-font-size-lead,
+    $theme-line-height-lead
+  );
+  font-weight: $theme-font-weight-normal;
+  max-width: measure($theme-measure-lead);
 }
 
 .usa-background-dark {
-  background-color: $color-gray-dark;
+  background-color: color('base-darker');
 
   p,
   span {
-    color: $color-white;
+    color: color('white');
   }
 
   a {
-    color: $color-gray-lighter;
+    color: color('base-lighter');
 
     &:hover {
-      color: $color-white;
+      color: color('white');
     }
   }
 }
 
+// TODO: How should we keep or adapt these?
+// Strong candidates for deprecation
```

Comment:
If you can find where this is being used and replace it with something in 2.0, then ok to remove.